examples/walkthrough/k8s201.md

```diff
@@ -71,9 +143,43 @@ spec:
     name: nginx
 ```
 
+#### Service Management
+
+If you have the source checked out or downloaded the pre-compiled release you should have an `examples` directory in the Kubernetes root directory. If not, download the [nginx service example definition](service.yaml).
+
+Create an nginx service:
+
+```sh
+kubectl create -f examples/walkthrough/service.yaml
+```
+
+List all services:
+
+```sh
+kubectl get services
+```
+
+Provided your pod & service IPs are accessible (depending on network setup), you should be able to access the service's http endpoint in a browser or curl on port 80.
```

Comment:
In general, these addresses aren't going to be accessible. Use a cloud load balancer or the apiserver proxy, as suggested above. Or, you could use load balancer type NodePort.